backend/cmm_helpers.ml

```diff
@@ -2653,6 +2668,50 @@ let transl_builtin name args dbg =
   | "caml_int64_ctz_unboxed_to_untagged" -> ctz Pint64 (one_arg name args) dbg
   | "caml_nativeint_ctz_unboxed_to_untagged" ->
     ctz Pnativeint (one_arg name args) dbg
+  (* Native_pointer: handled as unboxed nativeint *)
+  | "caml_ext_pointer_as_native_pointer" ->
+    Some(int_as_pointer (one_arg name args) dbg)
+  | "caml_native_pointer_load_immediate" ->
+    Some(Cop(Cload (Word_val, Mutable), args, dbg))
```

Comment:
This can be `Word_int`, same for the next case, I think.

Reply:
Fixed. 
It's fine to use Word_int here, not Word_val, even though we are storing a value (there is no untagging), because we know that the value passed to this function is an immediate, assuming that the external declaration is as in the `ocaml_intrinsics` library.